### PR TITLE
Replaced torch7 by onnx model in js_face_recognition dnn sample

### PR DESCRIPTION
**Pull request by alexlyulkov**
Changed face recognition model in js_face_recognition dnn sample: replaced torch7 model from https://github.com/pyannote/pyannote-data by ONNX model from https://github.com/opencv/opencv_zoo/tree/main/models/face_recognition_sface

### CODE REVIEW

File: samples/dnn/js_face_recognition.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
 //! [Get 128 floating points feature vector]
 function face2vec(face) {
-  var blob = cv.blobFromImage(face, 1.0 / 255, {width: 96, height: 96}, [0, 0, 0, 0], true, false)
+  var blob = cv.blobFromImage(face, 1.0, {width: 112, height: 112}, [0, 0, 0, 0], true, false)
```
Comment (alexlyulkov) [inline samples/dnn/js_face_recognition.html:43]: see https://github.com/opencv/opencv_zoo/blob/main/models/face_recognition_sface/demo.py#L37
https://github.com/opencv/opencv_zoo/blob/main/models/face_recognition_sface/sface.py#L15
https://github.com/opencv/opencv/blob/4.x/modules/objdetect/src/face_recognize.cpp#L45

File: samples/dnn/js_face_recognition.html [hunk 2]
```diff
@@ -71,15 +71,15 @@
...
   utils.createFileFromUrl('face_detector.prototxt', proto, () => {
     document.getElementById('status').innerHTML = 'Downloading face_detector.caffemodel';
     utils.createFileFromUrl('face_detector.caffemodel', weights, () => {
       document.getElementById('status').innerHTML = 'Downloading OpenFace model';
-      utils.createFileFromUrl('face_recognition.t7', recognModel, () => {
+      utils.createFileFromUrl('face_recognition_sface_2021dec.onnx', recognModel, () => {
         document.getElementById('status').innerHTML = '';
         netDet = cv.readNetFromCaffe('face_detector.prototxt', 'face_detector.caffemodel');
-        netRecogn = cv.readNetFromTorch('face_recognition.t7');
+        netRecogn = cv.readNetFromONNX('face_recognition_sface_2021dec.onnx');
```
Comment (dkurt) [inline samples/dnn/js_face_recognition.html:82]: `cv.readNet`

File: samples/dnn/js_face_recognition.html (hunk 1 above)
Comment (asmorkalov) [reply samples/dnn/js_face_recognition.html:43]: It makes sense to add reference to the model and parameters as comment to the code.

**Review by dkurt — APPROVED** [review]

**Review by asmorkalov — APPROVED** [review]
:+1:

**Pull request merged**